### PR TITLE
Bugfix 121: Detail page for an item without fulltext is broken

### PR DESCRIPTION
**Pull request by rivaldi8**
This connects to #121

### CODE REVIEW

**Comment by artlowel** [comment]
LGTM

File: src/app/item-page/simple/field-components/file-section/file-section.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ds-metadata-field-wrapper [label]="label | translate">
+<ds-metadata-field-wrapper *ngIf="(files | async).length > 0" [label]="label | translate">
```
Comment (artlowel) [inline src/app/item-page/simple/field-components/file-section/file-section.component.html]: It's probably safer to add a question mark after the right bracket, to ensure it doesn't break if `(files | async)` returns null or undefined for any reason.
```typescript
(files | async)?.length
```

Comment (rivaldi8) [reply src/app/item-page/simple/field-components/file-section/file-section.component.html]: Done!

**Comment by artlowel** [comment]
Looks good, merging

**Pull request merged**